config/src/lib.rs

```diff
@@ -68,7 +69,6 @@ pub struct ClientConfig {
     pub retry_policy: RetryWithBackoff,
 
     #[get]
-    #[builder(setter(into))]
```

Comment:
Is this being removed because it is redundant?

Reply:
Yes, it was removed because it duplicates the setting on the builder struct